src/client/Client.cc

```diff
@@ -6524,13 +6524,17 @@ int Client::_do_setattr(Inode *in, struct stat *attr, int mask, int uid, int gid
       mask &= ~(CEPH_SETATTR_MTIME|CEPH_SETATTR_ATIME);
     }
   }
-  if (!mask)
+  if (!mask) {
+    in->change_attr++;
     return 0;
+  }
 
 force_request:
   MetaRequest *req = new MetaRequest(CEPH_MDS_OP_SETATTR);
 
   filepath path;
+
+  in->change_attr++;
```

Comment:
So we're doing the local bump here, but then it gets bu,led again on the remote side, right?

If we want to do it locally on all conditions, should we do it above instead of duplicating between the remote and local-only code paths?